Temporary workaround for BC test due to schema parser changes

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32320 [JIT] Test passing custom class instance to bound method
* #32312 [JIT] Fix returning instance of custom class from method
* #32301 [JIT] Remove __torch__ from custom class qualname
* **#32324 Temporary workaround for BC test due to schema parser changes**

Differential Revision: [D19438085](https://our.internmc.facebook.com/intern/diff/D19438085)

Commit:
Temporary workaround for BC test due to schema parser changes

[ghstack-poisoned]

houseroad reviewed (approved):
Looks good to me.

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 951cdbca:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=951cdbcacec126b3a931913973b45be95eb6028b).

### 1 failure *not* recognized by patterns:
| Job | Step | Status |
| --- | --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>caffe2_onnx_py2_gcc5_ubuntu16_04_build</sup> | <sup>Build</sup> | New in PR |


---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "Temporary workaround for BC test due to schema parser changes"


Differential Revision: [D19438085](https://our.internmc.facebook.com/intern/diff/D19438085)

[ghstack-poisoned]

Commit:
Update on "Temporary workaround for BC test due to schema parser changes"


Differential Revision: [D19438085](https://our.internmc.facebook.com/intern/diff/D19438085)

[ghstack-poisoned]
